project/console.py

```diff
@@ -0,0 +1,67 @@
+import project
+from project.commands import ExecutionException
+
+command_names = ["graph_info", "create_and_save", "quit"]
```

Comment:
По заданию у вас должны быть следующие команды. (названия примерные)
```suggestion
command_names = ["get_graph_info", "create_two_cycles_graph", "save_to_dot", "quit"]
```

Reply:
Done